mono/eglib/glib.h

```diff
@@ -251,6 +251,10 @@ typedef guint32 gunichar;
 #define ALIGN_TO(val,align) ((((gssize)val) + ((align) - 1)) & ~((align) - 1))
 #endif
 
+#ifndef ALIGN_DOWN_TO
+#define ALIGN_DOWN_TO(val,align) (((gssize)val) & ~((align) - 1))
+#endif
+
 #ifndef ALIGN_PTR_TO
 #define ALIGN_PTR_TO(ptr,align) (gpointer)((((gssize)(ptr)) + (align - 1)) & (~(align - 1)))
 #endif
```

Comment:
Not sure what purpose these ifndefs serve. there are no other definitions of `ALIGN_DOWN_TO` in mono.

Reply:
I followed the style of the other macros around it, no strong feelings about it though (there are also no other definitions of `ALIGN_TO`/`ALIGN_PTR_TO`).